lib/include/jxl/decode.h

```diff
@@ -913,6 +953,144 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderSetJPEGBuffer(JxlDecoder* dec,
  */
 JXL_EXPORT size_t JxlDecoderReleaseJPEGBuffer(JxlDecoder* dec);
 
+/**
+ * Sets output buffer for box output codestream.
+ *
+ * The data is owned by the caller and may be used by the decoder until
+ * JxlDecoderReleaseBoxBuffer is called or the decoder is destroyed or reset so
+ * must be kept alive until then.
+ *
+ * @param dec decoder object
+ * @param data pointer to next bytes to write to
+ * @param size amount of bytes available starting from data
+ * @param get_raw_box_data whether to get raw, uncompressed box data in the case
+ * of a "brob" box. If false, automatically decompresses the box with brotli if
+ * it was compressed. This requires the decoder has been compiled with brotli
+ * support.
+ * @return JXL_DEC_ERROR if input was already set without releasing,
```

Comment:
What does "input was already set without releasing" mean?

Reply:
This was a copypasting error from "JxlDecoderSetInput", fixed